docs/docusaurus/docusaurus.config.js

```diff
@@ -78,80 +97,24 @@ module.exports = {
           dropdownActiveClassDisabled: true,
         },
         {
-          label: 'Product',
-          position: 'right',
-          items: [
-            {
-              label: 'GX CLOUD',
-              to: 'https://greatexpectations.io/gx-cloud',
-            },
-            {
-              label: 'GX OSS',
-              to: 'https://greatexpectations.io/gx-oss',
-            },
-          ],
+          label: 'HOME',
+          to: '/docs/home/',
         },
         {
-          label: 'Community',
-          position: 'right',
-          items: [
-            {
-              label: 'COMMUNITY HOME',
-              to: 'https://greatexpectations.io/community',
-            },
-            {
-              label: 'COMMUNITY FORUM',
-              to: 'https://discourse.greatexpectations.io/',
-            },
-            {
-              label: 'GITHUB',
-              to: 'https://github.com/great-expectations/great_expectations',
-            },
-            {
-              label: 'JOIN THE EMAIL LIST',
-              to: 'https://greatexpectations.io/newsletter',
-            },
-          ],
+          label: 'GX CLOUD',
+          to: '/docs/cloud/about_gx',
         },
         {
-          label: 'RESOURCES',
-          position: 'right',
-          items: [
-            {
-              label: 'INTEGRATIONS',
-              to: 'https://greatexpectations.io/integrations',
-            },
-            {
-              label: 'DOCUMENTATION',
-              to: 'https://docs.greatexpectations.io/docs/',
-            },
-            {
-              label: 'EXPECTATION GALLERY',
-              to: 'https://greatexpectations.io/expectations',
-            },
-            {
-              label: 'GREAT EXPECTATIONS BLOG',
-              to: 'https://greatexpectations.io/blog',
-            },
-            {
-              label: 'GREAT EXPECTATIONS CASE STUDIES',
-              to: 'https://greatexpectations.io/case-studies',
-            },
-          ],
+          label: 'GX OSS',
+          to: '/docs/guides/setup/get_started_lp',
         },
         {
-          label: 'Company',
-          position: 'right',
-          items: [
-            {
-              label: 'ABOUT US',
-              to: 'https://greatexpectations.io/company',
-            },
-            {
-              label: 'CAREERS',
-              to: 'https://jobs.greatexpectations.io/',
-            },
-          ],
+          label: 'GX OSS API',
+          to: '/docs/reference/api',
```

Comment:
@Rachel-Reverie  Does this fix the issue? This links to the top level api docs page https://docs.greatexpectations.io/docs/reference/api_reference
```suggestion
          to: '/docs/reference/api_reference',
```